rconweb/api/views.py

```diff
@@ -645,7 +679,8 @@ def get_connection_info(request):
 
 
 @csrf_exempt
-@login_required(True)
+@login_required()
+@permission_required("api.can_run_raw_commands", raise_exception=True)
```

Comment:
maybe you should do a super_admin group for that one

Reply:
Added an owner group in PR 250 that has that permission and removed it from the admin group.